interop-testing/src/main/java/io/grpc/testing/integration/TestServiceClient.java

```diff
@@ -162,6 +166,8 @@ void parseArgs(String[] args) {
           + "\n  --use_tls=true|false        Whether to use TLS. Default " + c.useTls
           + "\n  --use_alts=true|false       Whether to use ALTS. Enable ALTS will disable TLS."
           + "\n                              Default " + c.useAlts
+          + "\n  --custom_credentials_type   Custom credentials type to use. Default "
```

Comment:
This naming of "custom_credentials_type" come from grpc c core. Although I don't like this argument name, it is better to keep it so that interop tests can work in the future.